gtkwave.nsi

```diff
@@ -0,0 +1,179 @@
+  CreateDirectory "$SMPROGRAMS\${NAME_CAP}"
+  CreateShortCut "$SMPROGRAMS\${NAME_CAP}\Uninstall.lnk" "$INSTDIR\bin\uninst.exe" "" "$INSTDIR\uninst.exe" 0
+  CreateShortCut "$SMPROGRAMS\${NAME_CAP}\${NAME_CAP}.lnk" "$INSTDIR\bin\${NAME_CAP}.exe" "" "$INSTDIR\${NAME_CAP}.exe" 0
+SectionEnd
+
+Section "" ; (default section)
+
+  SetOutPath $INSTDIR
+  
+  File /oname=COPYING.txt COPYING
+  File README.md
+  
+  SetOutPath $INSTDIR\bin
+  File ${SYSROOT}\bin\libintl-8.dll
+  File ${SYSROOT}\bin\iconv.dll
+  File ${SYSROOT}\bin\libpcre-1.dll
+  File ${SYSROOT}\bin\libpcre2-8-0.dll
+  File ${SYSROOT}\bin\libgtk-3-0.dll
+  File ${SYSROOT}\bin\libgdk-3-0.dll
+  File ${SYSROOT}\bin\libssp-0.dll
+  File ${SYSROOT}\bin\libgdk_pixbuf-2.0-0.dll
+  File ${SYSROOT}\bin\libpixman-1-0.dll
+  File ${SYSROOT}\bin\libffi-8.dll
+  File ${SYSROOT}\bin\libgio-2.0-0.dll
+  File ${SYSROOT}\bin\libgnurx-0.dll
+  File ${SYSROOT}\bin\libcairo-2.dll
+  File ${SYSROOT}\bin\libcairo-gobject-2.dll
+  File ${SYSROOT}\bin\zlib1.dll
+  File ${SYSROOT}\bin\libglib-2.0-0.dll
+  File ${SYSROOT}\bin\libatk-1.0-0.dll
+  File ${SYSROOT}\bin\libgobject-2.0-0.dll
+  File ${SYSROOT}\bin\libgmodule-2.0-0.dll
+  File ${SYSROOT}\bin\libgthread-2.0-0.dll
+  File ${SYSROOT}\bin\libpango-1.0-0.dll
+  File ${SYSROOT}\bin\libfribidi-0.dll
+  File ${SYSROOT}\bin\libpangocairo-1.0-0.dll
+  File ${SYSROOT}\bin\libpangoft2-1.0-0.dll
+  File ${SYSROOT}\bin\libpangowin32-1.0-0.dll
+  File ${SYSROOT}\bin\libpng16-16.dll
+  File ${SYSROOT}\bin\libtiff-5.dll
+  File ${SYSROOT}\bin\libjpeg-62.dll
+  File ${SYSROOT}\bin\libfontconfig-1.dll
+  File ${SYSROOT}\bin\libxml2-2.dll
+  File ${SYSROOT}\bin\libfreetype-6.dll
+  File ${SYSROOT}\bin\gdk-pixbuf-query-loaders.exe
+  File ${SYSROOT}\bin\${LIBGCCDLL}
+  File ${SYSROOT}\bin\libstdc++-6.dll
+  File ${SYSROOT}\bin\libwinpthread-1.dll
+  File ${SYSROOT}\bin\libgfortran-*.dll
+  File ${SYSROOT}\bin\librsvg-2-2.dll
+  File ${SYSROOT}\bin\libcroco-0.6-3.dll
+  File ${SYSROOT}\bin\libxml2-2.dll
+  File ${SYSROOT}\bin\libquadmath-0.dll
+  File ${SYSROOT}\bin\libopenjp2.dll
+  File ${SYSROOT}\bin\libsqlite3-0.dll
+  File ${SYSROOT}\bin\libexpat-1.dll
+  File ${SYSROOT}\bin\libbz2-1.dll
+  #File ${SYSROOT}\bin\libgdkglext-win32-1.0-0.dll
+  #File ${SYSROOT}\bin\libgtkglext-win32-1.0-0.dll
+  File ${SYSROOT}\bin\glew32.dll
+  File ${SYSROOT}\bin\libzip-5.dll
+  File ${SYSROOT}\bin\libharfbuzz-0.dll
+  File ${SYSROOT}\bin\libharfbuzz-icu-0.dll
+  File ${SYSROOT}\bin\libharfbuzz-subset-0.dll
+  File ${SYSROOT}\bin\gspawn-win64-helper.exe
+  File ${SYSROOT}\bin\gdbus.exe
+  File ${SYSROOT}\bin\libsigc-2.0-0.dll
+  File ${SYSROOT}\bin\libmpfr-6.dll
+  File ${SYSROOT}\bin\libmpfr-6.dll
+  # Libepoxy in official package is broken, so we use an older working version!
+  # File ${SYSROOT}\bin\libepoxy-0.dll
+  File \usr\local\mingw64\bin\libepoxy-0.dll
+  File \usr\local\mingw64\bin\libfmt.dll
+  File \usr\local\mingw64\bin\libgmp-10.dll
+  File ${OUTDIR}\src\${NAME}.exe
```

Comment:
The command line only tools from the `src/helpers` directory should also be installed.